spinalcordtoolbox/utils/shell.py

```diff
@@ -244,7 +244,7 @@ def list_typecast_func(string):
     return list_typecast_func
 
 
-class Metavar(Enum):
+class Metavar(str, Enum):
```

Comment:
Source on this change: https://stackoverflow.com/a/58608362

Reply:
I also noticed that the example in ^ doesn't include a `__str__` method. However, excluding this method results in the following:

![image](https://user-images.githubusercontent.com/16181459/188907753-5563f38d-d35d-451c-ac74-471af4e45155.png)

Reply:
It looks like Python 3.11 has [`enum.StrEnum`](https://docs.python.org/3.11/library/enum.html#enum.StrEnum), so when we upgrade we should be able to use that (and it looks like [it won't need `__str__`](https://docs.python.org/3.11/library/enum.html#notes)? I'm not sure).